automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestParameterSetting.scala

```diff
@@ -257,7 +257,7 @@ class TestParameterSetting  extends ExtractionTest {
   val u5a = "For the purposes of model parameterisation the value of shoot_lag has been assumed to be around 40 o Cd."
   passingTest should s"extract the parameter setting(s) from u5a: ${u5a}" taggedAs(Somebody) in {
     val desired = Seq(
-      "value of shoot_lag" -> Seq("40") // fixme: ideally, should just be shoot_lag, but this tests the var-assigned rule
+      "shoot_lag" -> Seq("40") // fixme: ideally, should just be shoot_lag, but this tests the var-assigned rule
```

Comment:
Seems like you addressed this fixme! Please delete the note before merging.

Reply:
good eye! deleted